torque-core/src/main/kotlin/com/workday/torque/LogcatRecorder.kt

```diff
@@ -52,10 +62,11 @@ class LogcatRecorder(private val logcatFileIO: LogcatFileIO){
                 .apply { appendNewLogLine(newLine) }
     }
 
+    private fun TestLogcat.isMatchingFinishTestLogcat(newLine: String): Boolean {
+        return testDetails == newLine.parseTestDetails()!!
```

Comment:
Is !! needed here? This might lead to another crash.

Reply:
good point! let me get rid of all of them in this class